src/librustc_mir/shim.rs

```diff
@@ -213,10 +221,10 @@ fn build_drop_shim<'a, 'tcx>(tcx: ty::TyCtxt<'a, 'tcx, 'tcx>,
 }
 
 pub struct DropShimElaborator<'a, 'tcx: 'a> {
```

Comment:
Its location makes sense to me, as it's still used to create a drop shim (just in generator.rs).